code/__DEFINES/beestation.dm

```diff
@@ -0,0 +1,3 @@
+#define COMSIG_MOB_ATTACK_HAND_TURF "mob_attack_hand_turf"	//Called on mob, params: turf
+
+#define MOVESPEED_ID_SWIMMING "SWIMMING_SPEED_MOD"
```

Comment:
This file exists on NSV to prevent merge conflicts. We don't have to worry about that, so put it in the relevent file for defines